admin.html

```diff
@@ -74,9 +78,26 @@ <h2 class="mdl-card__title-text">Create a new DoWhop</h2>
                   <span class="mdl-tooltip" for="doer-description">Enter multiple doers, seperated, by, Commas</span>
                 </div>
                 <div class="mdl-card__actions">
-                  <input id="dowhop-image-capture1" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
-                  <input id="dowhop-image-capture2" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
-                  <input id="dowhop-image-capture3" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
+                  <div id="create-outer-div">
```

Comment:
Thanks @mike-mjk! Yes, that sounds good. For image-1, etc you might have to add more to make sure they stay unique if they are IDs. 
We inherited some of this code so w'Il keep finding stuff to clean.